CMakeLists.txt

```diff
@@ -167,6 +167,27 @@ if(APPLE)
     endif()
 endif()
 
+if(OPTION_PACKAGE_DEPENDENCIES)
+    if(NOT UNIX)
+        message(FATAL_ERROR "Can only package dependencies on Linux")
+    endif()
+    if(DEFAULT_OPTION_INSTALL_FHS)
```

Comment:
Should be `OPTION_INSTALL_FHS` I think as it can be manually set to something different than default